### PR TITLE
PXS-991 Mapbox exit confirmation

### PR DESCRIPTION
**Pull request by ToufiPF**
Display a confirmation dialog for when user inadvertently presses the back arrow. If user clicks on "yes", then they are redirected to the FeedbackActivity (as if they clicked on the ❎ button to exit the tour). Otherwise, they stay on the mapbox activity.

| Style 1 (actual) | Style 2 |
| -------- | -------- |
|  |  |

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -52,10 +56,10 @@
     <!-- Mapbox buttons-->
     <string name="mapbox_2d_button_text">2D</string>
     <string name="mapbox_3d_button_text">3D</string>
-    <string name="follow_user_button_text">Centrer sur l\'utilisateur</string>
-    <string name="close_poi_button_content_description">Quitter le point d\'intérêt</string>
+    <string name="mapbox_follow_user_button_text">Centrer sur l\'utilisateur</string>
+    <string name="mapbox_exit_confirmation">Êtes-vous sûr (sure) de vouloir quitter ?</string>
```
Comment (amomo25) [inline app/src/main/res/values-fr/strings.xml:60]: I think you need to add a circumflex accent on "sure"

**Review by amomo25 — COMMENTED** [review]
Just a small spelling mistake on the text, the rest looks fine to me

**Review by amomo25 — APPROVED** [review]

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -52,10 +56,10 @@
     <!-- Mapbox buttons-->
     <string name="mapbox_2d_button_text">2D</string>
     <string name="mapbox_3d_button_text">3D</string>
-    <string name="follow_user_button_text">Centrer sur l\'utilisateur</string>
-    <string name="close_poi_button_content_description">Quitter le point d\'intérêt</string>
+    <string name="mapbox_follow_user_button_text">Centrer sur l\'utilisateur</string>
+    <string name="mapbox_exit_confirmation">Êtes-vous sûr(e) de vouloir quitter ?</string>
```
Comment (ToufiPF) [inline app/src/main/res/values-fr/strings.xml:60]: Is it better now?

Comment (amomo25) [reply app/src/main/res/values-fr/strings.xml:60]: Yes, perfect

**Pull request merged**